samples/snippets/src/test/java/aiplatform/ExportModelTabularClassificationSampleTest.java

```diff
@@ -32,8 +32,7 @@
 public class ExportModelTabularClassificationSampleTest {
 
   private static final String PROJECT = System.getenv("UCAIP_PROJECT_ID");
-  private static final String MODEL_ID =
-      System.getenv("EXPORT_MODEL_TABLES_CLASSIFICATION_MODEL_ID");
+  private static final String MODEL_ID = "6036688272397172736";

```

Comment:
same comment as above. please reconsider the hardcoding.

Reply:
Acknowledged.